docs/source/checkpoints/dist_ckpt.rst

```diff
@@ -0,0 +1,392 @@
+Distributed checkpoints
+=======================
+
+This guide provides details about the distributed checkpoints format from Megatron-Core.
+
+
+Introduction
+------------
+
+Model parallel training requires parallelism-aware checkpointing.
+Megatron-Core provides a checkpointing library capable of handling all types of parallelisms used in LLMs training.
+Although distributed checkpointing library is targeted at Megatron-Core model, it can be used with other models as well, provided an appropriate integration.
```

Comment:
While the distributed checkpointing library is designed for the Megatron Core model, it can also be used with other models, as long as proper integration is implemented.